### ISSUE TITLE
setOrder() bug for ordering class members

### ISSUE THREAD

**Issue by gregjacobs** [dsherret/ts-simple-ast#244]
Hey David,

I discovered a bug in `setOrder()` when ordering class members.

I'm basically trying to move a class's constructor below all of the class's properties. But for some reason, any argument >4 passed to `setOrder()` gives me the following error:

```
Error: Argument Error (order): Range is 0 to 4, but 6 was provided.
```

However, in my example below, there are clearly there are more than 4 class members. Give it a try:

```ts
import Ast from 'ts-simple-ast';

const ast = new Ast();
const sourceFile = ast.createSourceFile( 'test.ts', `
    class Something {
        constructor() {}
		
        propA = 1;
        propB = 2;
        propC = 3;
        propD = 4;
        propE = 5;
        propF = 6;
        propG = 7;
    }
` );

const myClass = sourceFile.getClasses()[ 0 ];
const ctor = myClass.getConstructors()[ 0 ];

ctor.setOrder( 6 );  // <-- Any number greater than 4 here throws an error

console.log( sourceFile.getFullText() );
```

Why any number greater than `4` specifically? Dunno :) I encountered the same error on my Angular migrator too, which also gave an error for any index > `4`, but with completely different source text. Came up with the above minimal reproduction separately.

This is with the latest btw: 6.10.0

**Comment by gregjacobs** [comment]
Full stack trace if you're interested btw:

```
(14:26:09) ~/dev/ts-simple-ast-test $ ts-node test-order.ts 

/Users/gjacobs/dev/ts-simple-ast-test/node_modules/ts-simple-ast/dist/errors/BaseError.js:16
        var _this = _super.call(this, message) || this;
                           ^
Error: Argument Error (order): Range is 0 to 4, but 6 was provided.
    at ArgumentOutOfRangeError.BaseError [as constructor] (/Users/gjacobs/dev/ts-simple-ast-test/node_modules/ts-simple-ast/dist/errors/BaseError.js:16:28)
    at ArgumentOutOfRangeError.ArgumentError [as constructor] (/Users/gjacobs/dev/ts-simple-ast-test/node_modules/ts-simple-ast/dist/errors/ArgumentError.js:18:28)
    at new ArgumentOutOfRangeError (/Users/gjacobs/dev/ts-simple-ast-test/node_modules/ts-simple-ast/dist/errors/ArgumentOutOfRangeError.js:17:28)
    at Object.throwIfOutOfRange (/Users/gjacobs/dev/ts-simple-ast-test/node_modules/ts-simple-ast/dist/errors/helpers.js:53:15)
    at ConstructorDeclaration.ChildOrderableNode.class_1.setOrder (/Users/gjacobs/dev/ts-simple-ast-test/node_modules/ts-simple-ast/dist/compiler/base/ChildOrderableNode.js:23:20)
    at Object.<anonymous> (/Users/gjacobs/dev/ts-simple-ast-test/test-order.ts:21:6)
    at Module._compile (module.js:635:30)
    at Module.m._compile (/usr/local/lib/node_modules/ts-node/src/index.ts:422:23)
    at Module._extensions..js (module.js:646:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/usr/local/lib/node_modules/ts-node/src/index.ts:425:12)
```

**Comment by gregjacobs** [comment]
Thanks @dsherret, that did the trick! 

Would be great if you could add that in the next version! Can't ask everyone at work to modify their node_modules :) hehe

**Comment by dsherret** [comment]
As much as I wanted to add those instructions to readme.md, this is fixed in 6.11.1 :)

Thanks again for reporting the bugs you find!

**Comment by gregjacobs** [comment]
Hahaha, you could add those instructions ;) lol

Thanks for fixing!
